test/extended/prometheus/prometheus.go

```diff
@@ -211,9 +211,8 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
 
 			tests := map[string][]metricTest{
-				// should be checking there is no more than 1 alerts firing.
-				// Checking for specific alert is done in "should have a Watchdog alert in firing state".
-				`ALERTS{alertstate="firing"}`: {metricTest{greaterThanEqual: false, value: 2}},
+				// Checking Watchdog alert state is done in "should have a Watchdog alert in firing state".
+				`ALERTS{alertname!="Watchdog",alertstate="firing"}`: {metricTest{greaterThanEqual: true, value: 1}},
```

Comment:
{greaterThanEqual: true, value: 1} will this output success on 1? I thought if there is 1 for particular alert record it should fail, I don't know much about metrics though

Reply:
like the alerts outputted for me are:
```
ALERTS{alertname="UsingDeprecatedAPIExtensionsV1Beta1",alertstate="firing",client="cluster-policy-controller/v0.0.0 (linux/amd64) kubernetes/$Format",code="0",component="apiserver",contentType="application/vnd.kubernetes.protobuf;stream=watch",endpoint="https",group="extensions",instance="10.0.138.54:6443",job="apiserver",namespace="default",resource="daemonsets",scope="cluster",service="kubernetes",severity="warning",verb="WATCH",version="v1beta1"}
| 1 @1571823124.618   1 @1571823154.618   1 @1571823184.618   1 @1571823214.618  ....|
```